celery/canvas.py

```diff
@@ -1142,6 +1149,7 @@ def _freeze_unroll(self, new_tasks, group_id, chord, root_id, parent_id):
         # pylint: disable=redefined-outer-name
         #   XXX chord is also a class in outer scope.
         stack = deque(self.tasks)
+        i = 0
```

Comment:
Perhaps minor, but I think we could also name the variable `group_index` instead of `i`.

Reply:
Done.